basics/data-import/pinot-stream-ingestion/amazon-kinesis.md

```diff
@@ -55,9 +61,9 @@ Kinesis supports authentication using the [DefaultCredentialsProviderChain](http
 * Credentials delivered through the Amazon EC2 container service if `AWS_CONTAINER_CREDENTIALS_RELATIVE_URI` environment variable is set and security manager has permission to access the variable,
 * Instance profile credentials delivered through the Amazon EC2 metadata service
 
-You can also specify the accessKey and secretKey using the properties. However, this method is not secure and should be used only for POC setups. You can also specify other aws fields such as AWS\_SESSION\_TOKEN as environment variables and config and it will work.
+You can also specify the `accessKey` and `secretKey` using the properties. However, this method is not secure and should be used only for POC setups. You can also specify other AWS fields such as AWS\_SESSION\_TOKEN as environment variables and config and it will work.
```

Comment:
```suggestion
Although you can specify the `accessKey` and `secretKey` in properties, we don't recommend this unsecure method. We recommend using only for POC setups. You can also specify other AWS fields such as AWS\_SESSION\_TOKEN as environment variables.
```

Reply:
where is "properties"?--can we include a link to? 

what are POC setups (non-production/proof of concept)?